scripts/xtensa-build-zephyr.py

```diff
@@ -532,8 +540,11 @@ def build_platforms():
 
 		platform_dict = [x for x in platform_list if x["name"] == platform][0]
 		xtensa_tools_root_dir = os.getenv("XTENSA_TOOLS_ROOT")
-		# when XTENSA_TOOLS_ROOT environmental variable is set,
-		# use user installed Xtensa tools not Zephyr SDK
+
+		# for some reason CI does not set XTENSA_TOOLS_ROOT while using xtensa toolchain...
+		if not xtensa_tools_root_dir and os.environ.get("XTENSA_INSTALL_PATH"):
+			# and has specific XTENSA_INSTALL_PATH set instead...
+			xtensa_tools_root_dir = pathlib.Path(os.environ.get("XTENSA_INSTALL_PATH"), "..", "..")
```

Comment:
Note for other reviewers: this is part of the temporary, DO NOT MERGE commit. Do not review. It looked "too good" so I wasted a bit of time reviewing it, next time please add some uglier comments :-)